src/js/components/Data/DataForm.js

```diff
@@ -229,6 +229,23 @@ const normalizeValue = (nextValue, prevValue, views) => {
   return result;
 };
 
+// 300ms was chosen empirically as a reasonable default
+const DEBOUNCE_TIMEOUT = 300;
```

Comment:
If needed in the future, is there an approach where we can we allow the caller to specify the debounce value if desired? I believe there is, but just confirming.

Reply:
Theme maybe?

Reply:
> Theme maybe?

As a default, I think the theme could make sense. Was wondering about specific instances where the performance of the API being interacted with might also be a consideration and the implementor might want control to tune for that instance. Thoughts @MikeKingdom ?

Reply:
Not saying this needs to be implemented in this PR, rather just a line of sight to how we might approach if needed.

Reply:
Yeah definitely valid. Might be worth a prop

Reply:
Yeah I think there are a variety of approaches we could take to allow caller to drive (whether via prop, or theme, etc.). And then code would just modify where default value is used to something like `propValue || DEBOUNCE_TIMEOUT`